python/sqlflow_submitter/tensorflow/train.py

```diff
@@ -40,10 +40,49 @@
     tf.logging.set_verbosity(tf.logging.ERROR)
     TF_VERSION_2 = False
 
-# FLAGS only available for TF 1.x to support PAI TF training
-if not TF_VERSION_2:
-    tf.app.flags.DEFINE_string("tables", "", "tables info")
-    FLAGS = tf.app.flags.FLAGS
+FLAGS = None
+
+# ----------------- For PAI distributed training -----------------
+def define_tf_flags():
+    global FLAGS
+    if not TF_VERSION_2:
```

Comment:
It seems PAI will fill these flags automatically, how about adding a comment for this behavior?

Reply:
`# ----------------- For PAI distributed training -----------------` is just it.

Reply:
`# ----------------- For PAI distributed training -----------------` is just it.